app/helpers/brexit_checker_helper.rb

```diff
@@ -38,10 +38,9 @@ def format_question_option(option, criteria_keys)
       hint_text: option.hint_text }
   end
 
-  def next_question_index(all_questions:, criteria_keys: [], previous_question_index: 0)
+  def next_question(all_questions:, criteria_keys: [], previous_question_index: 0)
     available_questions = all_questions[previous_question_index..] || []
```

Comment:
It still looks like we are relying on the order of the questions in the array at this point. To move away from that, we’ll need to reorder the questions by their index.

Reply:
🤦‍♂ - yup!